vendor/kubesphere.io/api

```diff
@@ -1 +0,0 @@
-../../staging/src/kubesphere.io/api
```

Comment:
Do not remove this symlink, use `hack/update-vendor.sh` to update the vendor directory.

Reply:
Done.